Gemfile

```diff
@@ -2,14 +2,17 @@ source "https://rubygems.org"
 
 gemspec
 
+# for generating i18n files, gettext > 3.0 dropped ruby 1.8 support
```

Comment:
The comment was brought over from katello's Gemfile. Foreman's has the same line but no comment. Suggestions?